config/routes.rb

```diff
@@ -29,4 +29,5 @@
   patch '/favorites/:pet_id', to: 'favorites#update'
   get '/favorites', to: 'favorites#index'
   delete '/favorites/:pet_id', to: 'favorites#destroy'
+  delete '/favorites/', to: 'favorites#destroy'
```

Comment:
the idea is that the URI is '/favorites/' because this route is destroy all favorites. uses the destroy action from favorites controller since it seems making up new actions is not convention and not RESTful. maybe making a separate controller would be better, ie all_favorites

Reply:
this is confusing to me!